src/codeu/chat/util/Interest.java

```diff
@@ -0,0 +1,59 @@
+package codeu.chat.util;
+
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.server.Model;
+import codeu.chat.util.store.Store;
+import codeu.chat.util.store.StoreAccessor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Interest {
```

Comment:
'Interest' doesn't quite describe what this class does because it's not a class that represents interests but rather a class used to store all interests. It should be 'InterestStore' at least.